### ISSUE TITLE
command not working

### ISSUE THREAD

**Issue by Pallav277** [duckietown/duckietown-shell-commands#324]
I have tried using following command,
"dts init_sd_card --hostname myrobot --type duckiebot --configuration DB21M --wifi pallav:0987654321"

But it is not initializing the download file eventhough internet is provided.
![Screenshot from 2022-06-03 14-43-42](https://user-images.githubusercontent.com/58349039/171826214-3e4525ac-f97b-4fc5-a919-8e1af31324f6.png)

**Comment by Pallav277** [comment]
I have even tried by reusing the zip file of Jetson Jetpack of 2GB version from the following link:
https://developer.nvidia.com/embedded/l4t/r32_release_v7.1/jp_4.6.1_b110_sd_card/jetson_nano_2gb/jetson-nano-2gb-jp461-sd-card-image.zip

And it gives Input/Output Error by os.fsync() function in dd.py file as follows:
![Screenshot from 2022-06-03 14-40-14](https://user-images.githubusercontent.com/58349039/171827367-484e3b73-d011-4be9-a0d6-db3024dd40e3.png)

**Comment by Pallav277** [comment]
I have also tried by commenting the os.fsync() in dd.py file and then omitting the download steps by --no-steps. It flashes the card but the verification steps fails. And Mismatch in range position Error pops up.
![Screenshot from 2022-06-03 15-13-46](https://user-images.githubusercontent.com/58349039/171831113-a76f474e-c326-4ab1-9ad1-e7633ebb85c8.png)

After that I also skipped the verification steps then following error occurs,
![Screenshot from 2022-06-03 15-26-19](https://user-images.githubusercontent.com/58349039/171833332-7a9be17b-ec34-414d-b157-90bd33c7e894.png)

**Comment by Pallav277** [comment]
I have also tried by commenting the os.fsync() in dd.py file and then omitting the download steps by --no-steps. It flashes the card but the verification steps fails. And Mismatch in range position Error pops up.
![Screenshot from 2022-06-03 15-13-46](https://user-images.githubusercontent.com/58349039/171831113-a76f474e-c326-4ab1-9ad1-e7633ebb85c8.png)

After that I also skipped the verification steps then following error occurs,
![Screenshot from 2022-06-03 15-26-19](https://user-images.githubusercontent.com/58349039/171833332-7a9be17b-ec34-414d-b157-90bd33c7e894.png)

**Comment by jesima31** [comment]
@Pallav277 Did you find the solution?

**Comment by jasonhu5** [comment]
@jesima31 @Pallav277 Is this still a problem? Is it that the download progress does not forward?

**Comment by jesima31** [comment]
Yes, the download was really slow. It took hours to download with a good internet connection but now not able to download. [https://github.com/duckietown/duckietown-shell-commands/issues/357](url)

**Comment by Pallav277** [comment]
Ohk, great. Thanks !!

**Comment by afdaniele** [comment]
@Pallav277 @jesima31 which countries are you connecting from?

**Comment by jesima31** [comment]
I am connecting from SG.

**Comment by BEmran** [comment]
Is there a solution to this issue. I am facing the same problem and I have tried different SD cards.
